tests/cli/test_cli_arguments.py

```diff
@@ -5,17 +5,13 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import subprocess
-import unittest
+import re
 
-
-class TestCLIArguments(unittest.TestCase):
+def test_unrecognized_argument():
     """Test CLI entry script for arguments."""
```

Comment:
You can remove this doc string. The test name is clear enough. It also applies to other methods below.